apps/frontend/package.json

```diff
@@ -17,14 +17,19 @@
     "@radix-ui/react-popover": "^1.0.7",
     "@radix-ui/react-select": "^2.0.0",
     "@radix-ui/react-tabs": "^1.0.4",
+    "@radix-ui/react-tooltip": "^1.0.7",
     "@react-oauth/google": "^0.11.1",
     "@tremor/react": "^3.9.2",
+    "@types/react-color": "^3.0.10",
```

Comment:
@kaf-lamed-beyt This should be in dev dependencies.

Reply:
Okay. moving it now.